public/js/common/cookie_handler.js

```diff
@@ -22,12 +22,12 @@ function getCookie(cookieName) {
     'use strict';
 
     var name = cookieName + '=';
-    var cookie = document.cookie.split(';');
-    for (var i = 0; i < cookie.length; i++) {
-      var c = cookie[i].trim();
-      if (c.indexOf(name) === 0) {
-            return c.substring(name.length, c.length);
-      }
+    var cookies = document.cookie.split(';');
+    for (var i = 0; i < cookies.length; i++) {
+        var cookie = cookies[i].trim();
+        if (cookie.indexOf(name) === 0) {
+            return cookie.substring(name.length, cookie.length);
+        }
     }
     return '';
 }
```

Comment:
I've gone through the JS files and added newlines to the end of files that didn't already have one.